tests/rptest/tests/crl_test.py

```diff
@@ -51,6 +53,11 @@ def create_service_client_cert(self, _, name):
                                  common_name=name))
         return self.service_client_certs[-1]
 
+    def find_broker_cert(self, node: ClusterNode) -> Optional[tls.Certificate]:
+        return next(
+            (c for c in self.broker_certs if node.account.hostname in c.crt),
```

Comment:
It;s not obvious to me that `node.name` and `node.account.hostname` are the same in every environment this runs in, but that assumption does seem to run throughout the test.

It might be simpler to make `self.broker_certs` a map.